website/static/js/institutionSignIn.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+var ko = require('knockout');
+
+var $osf = require('js/osfHelpers');
+
+var ViewModel = function() {
+    var self = this;
+    self.instNames = ko.observableArray([]);
+    self.selectedInst = ko.observable();
+    self.insts = {};
+    $osf.ajaxJSON(
```

Comment:
As mentioned in a previous comment, avoid making HTTP requests upon instantiation.